zetta_utils/tensor_ops/common.py

```diff
@@ -388,7 +388,7 @@ def interpolate(  # pylint: disable=too-many-locals
         allow_slice_rounding=allow_slice_rounding,
     )
 
-    if mode == "segmentation" and (
+    if mode in ("segmentation", "img", "bilinear") and (
```

Comment:
It's not going to break things, but `tinybrain` has no C implementation for signed integer type averaging (e.g. image encodings). If I read it correctly, it will convert such cases to fp32 and use numpy based interpolation. Probably slower than torch+CUDA

Reply:
Addressed via https://github.com/seung-lab/tinybrain/issues/22